databricks/koalas/tests/test_groupby.py

```diff
@@ -2245,3 +2245,104 @@ def test_is_multi_agg_with_relabel():
 
         assert is_multi_agg_with_relabel(a="max") is False
         assert is_multi_agg_with_relabel(a_min=("a", "max"), a_max=("a", "min")) is True
+
+    def test_get_group(self):
+        pdf = pd.DataFrame(
+            [
+                ("falcon", "bird", 389.0),
+                ("parrot", "bird", 24.0),
+                ("lion", "mammal", 80.5),
+                ("monkey", "mammal", np.nan),
+            ],
+            columns=["name", "class", "max_speed"],
+            index=[0, 2, 3, 1],
+        )
+        pdf.columns.name = "Koalas"
+        kdf = ks.from_pandas(pdf)
+
+        self.assert_eq(
+            kdf.groupby("class").get_group("bird"), pdf.groupby("class").get_group("bird"),
+        )
+        self.assert_eq(
+            kdf.groupby("class")["name"].get_group("mammal"),
+            pdf.groupby("class")["name"].get_group("mammal"),
+        )
+        self.assert_eq(
+            kdf.groupby("class")[["name"]].get_group("mammal"),
+            pdf.groupby("class")[["name"]].get_group("mammal"),
+        )
+        self.assert_eq(
+            kdf.groupby(["class", "name"]).get_group(("mammal", "lion")),
+            pdf.groupby(["class", "name"]).get_group(("mammal", "lion")),
+        )
```

Comment:
What if `kdf.groupby(["class", "name"]).get_group("mammal")`?

Reply:
What about `kdf.groupby('max_speed').get_group(80.5)`?

Reply:
first one raises `ValueError: must supply a tuple to get_group with multiple grouping keys`.

second one I'll fix it. 😿

Reply:
Shall we raise the same exception then?

Reply:
Sure.

Btw, I think it's already tested in `test_groupby.py`, line 2300 ?